doc/contribute-to-core-lightning/testing.md

```diff
@@ -41,6 +41,10 @@ There are four kinds of tests:
 
   `PYTHONPATH=contrib/pyln-client:contrib/pyln-testing:contrib/pyln-proto:contrib/pyln-grpc-proto py.test -v tests/`
 
+  Or alternatively, via the `Makefile`, by setting `PYTEST_TESTS`:
```

Comment:
This should be added after the next paragraph since the next paragraph is related to PYTHONPATH setup.

Reply:
Good catch. This has been moved.